prepare.js

```diff
@@ -6,19 +6,48 @@ function elapsed(hrtime) {
   return seconds;
 }
 
+function getHashOfI18nFiles() {
+  return execSync('cat config/locales/* | sha1sum').toString().split('-')[0].trim();
+}
+
 let start = process.hrtime();
 const execSync = require('child_process').execSync;
+const exec = require('child_process').exec;
+
 const config = require('./config');
+const fs = require('fs');
+
+let fullRebuild = false;
+try {
```

Comment:
Can you add comments explaining how this works? The code is pretty clear, but this is the kind of things that developers will appreciate not needing to read any code to understand.